test/index.js

```diff
@@ -763,6 +927,7 @@ function startLanguageServer(t, serverFilePath, cwd = '.') {
   const proc = spawn(
     'node',
     [
+      // '--inspect-brk=9229',
```

Comment:
funky

Reply:
Also very useful sometimes!

Of course I’ll remove it before lifting the draft status.